src/gui/updater/updater.cpp

```diff
@@ -165,7 +164,7 @@ qint64 Updater::Helper::stringVersionToInt(const QString &version)
 
 QString Updater::clientVersion()
 {
-    return QString::fromLatin1(MIRALL_STRINGIFY(MIRALL_VERSION_FULL));
+    return QStringLiteral("%1.%2.%3").arg(QString::number(MIRALL_VERSION_MAJOR), QString::number(MIRALL_VERSION_MINOR), QString::number(MIRALL_VERSION_PATCH));
```

Comment:
Ah, this *intentionally* removes the `BUILD_NUMBER`, right? Can you add a comment why it's not part of the version anymore or why it should not be readded?

Reply:
I did not 😱